plugins/micromega/g_zify.mlg

```diff
@@ -28,9 +28,8 @@ VERNAC COMMAND EXTEND DECLAREINJECTION CLASSIFIED AS SIDEFF
 | ["Add" "PropOp"    constr(t) ] -> { Zify.PropBinOp.register t }
 | ["Add" "PropBinOp"    constr(t) ] -> { Zify.PropBinOp.register t }
 | ["Add" "PropUOp"   constr(t) ] -> { Zify.PropUnOp.register t }
-| ["Add" "Spec"      constr(t) ] -> { Zify.Spec.register t }
```

Comment:
Shouldn't this deprecation of `Add Spec` be documented in CHANGELOG?

Reply:
Actually `micromega.rst` has not been updated accordingly.

Reply:
The `Add xxx` are currently undocumented...

Reply:
Sure. Actually what I saw in the documentation was `Show Zify ...` commands.